File: app/src/main/java/com/example/systemperingatan/SQLite/GeofenceContract.java
package com.example.systemperingatan.SQLite;

import android.provider.BaseColumns;
public class GeofenceContract {

    private GeofenceContract() {

    }


    public static class GeofenceEntry implements BaseColumns {
        public static final String TABLE_NAME = "geofences";
        public static final String COLUMN_NAME_KEY = "keys";
        public static final String COLUMN_NAME_LAT = "lat";
        public static final String COLUMN_NAME_LNG = "lng";
        public static final String COLUMN_NAME_EXPIRES = "expires";
    }
}

File: app/src/main/java/com/example/systemperingatan/SQLite/GeofenceContract.kt
package com.example.systemperingatan.SQLite

import android.provider.BaseColumns

object GeofenceContract {


    class GeofenceEntry : BaseColumns {
        companion object {
            val _ID: String = "id"
            val TABLE_NAME = "geofences"
            val COLUMN_NAME_KEY = "keys"
            val COLUMN_NAME_LAT = "lat"
            val COLUMN_NAME_LNG = "lng"
            val COLUMN_NAME_EXPIRES = "expires"

        }
    }
}
